### PR TITLE
For Ember Times No. 123: Add Writeup For Ember Fest Videos

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-11-08-the-ember-times-issue-123.md [hunk 1]
```diff
@@ -105,19 +106,19 @@ Sign up for a reminder on the [EmberConf CFP Brainstorm website](https://emberco
...
 
 ---
 
 ## [Contributors' Corner 👏](https://guides.emberjs.com/release/contributing/repositories/)
 
-<p>This week we'd like to thank our siblings for their contributions to Ember and related repositories! 💖</p>
+<p>This week we'd like to thank <a href="https://github.com/rwjblue" target="gh-user">@rwjblue</a>, <a href="https://github.com/Gaurav0" target="gh-user">@Gaurav0</a>, <a href="https://github.com/igorT" target="gh-user">@igorT</a>, <a href="https://github.com/runspired" target="gh-user">@runspired</a>, <a href="https://github.com/sivakumar-kailasam" target="gh-user">@sivakumar-kailasam</a>, <a href="https://github.com/addamh" target="gh-user">@addamh</a>, <a href="https://github.com/chancancode" target="gh-user">@chancancode</a>, <a href="https://github.com/kellyselden" target="gh-user">@kellyselden</a>, <a href="https://github.com/SparshithNR" target="gh-user">@SparshithNR</a>, <a href="https://github.com/abel-n" target="gh-user">@abel-n</a>, <a href="https://github.com/apps/github-actions" target="gh-user">@github-actions[bot]</a> and <a href="https://github.com/jenweber" target="gh-user">@jenweber</a> for their contributions to Ember and related repositories! 💖</p>
```
Comment (ijlee2) [inline source/2019-11-08-the-ember-times-issue-123.md]: I think I had seen `github-actions[bot]` some time back. I was wondering if we meant to include this as a contributor?

**Review by ijlee2 — APPROVED** [review]
The writeup for the EmberFest looks great. Thanks!

File: source/2019-11-08-the-ember-times-issue-123.md (hunk 1 above)
Comment (jayjayjpg) [reply source/2019-11-08-the-ember-times-issue-123.md]: That's right, I can remove that one - good catch!

**Comment by jayjayjpg** [comment]
Thank you 🙇‍♀

**Pull request merged**